erisdb/pipe/transactor.go

```diff
@@ -212,92 +212,6 @@ func (this *transactor) TransactAndHold(privKey, address, data []byte, gasLimit,
 	return ret, rErr
 }
 
-func (this *transactor) Send(privKey, toAddress []byte, amount int64) (*Receipt, error) {
```

Comment:
@ebuchman @NodeGuy @silasdavis can someone briefly remind me why we are deprecating the Send and SendAndHold methods for eris-db?